Makefile

```diff
@@ -31,3 +31,14 @@ fmt:
 # `make -k test-all docker-compose-down`, tears down the Docker services after running all the tests.
 test-all: docker-compose-up
 	QUICKWIT_ENV=local cargo test --all-features
+
+# This will build and push all custom cross images for cross-compilation.
+# You will need to login into dockerhub with `quickwitinc` account
+IMAGE_TAGS = x86_64-unknown-linux-gnu aarch64-unknown-linux-gnu x86_64-unknown-linux-musl aarch64-unknown-linux-musl 
+
+.PHONY: cross-images
+cross-images:
+	for tag in ${IMAGE_TAGS}; do \
+		docker build --tag quickwitinc/builder-$$tag --file ./cross-images/$$tag.dockerfile ./cross-images; \
```

Comment:
Who owns this DockerHub account? You?

Reply:
We own it this is where our docker images are pushed

Reply:
There's also:
https://hub.docker.com/orgs/quickwit

We can figure that out later but we need to consolidate around one account.

Reply:
I can definetly route all our images to this account if this is the official one. before merging this.
We just need someone with access to change the dockerhub access token on the github repository.